mars/scheduler/graph.py

```diff
@@ -654,6 +660,25 @@ def free_tensor_data(self, tensor_key):
             op_ref = self.ctx.actor_ref(op_uid, address=scheduler_addr)
             op_ref.free_data(_tell=True)
 
+    def get_tensor_merge_graph(self, tensor_key):
```

Comment:
``get_tensor_merge_graph`` may be renamed to ``build_tensor_merge_graph`` as the merge graph did not exist previously.